platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/internal/RichTextEditorStyleExt.kt

```diff
@@ -89,13 +91,23 @@ internal fun TextStyle.rememberTypeface(): State<Typeface> {
 
 internal fun TextView.applyStyleInCompose(style: RichTextEditorStyle) {
     setTextColor(style.text.color.toArgb())
-    val lineHeightInPx = TypedValue.applyDimension(
-        TypedValue.COMPLEX_UNIT_DIP,
-        style.text.lineHeight.value,
-        context.resources.displayMetrics
-    )
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-        lineHeight = lineHeightInPx.roundToInt()
+    if (style.text.lineHeight.isSpecified && style.text.lineHeight.value > 0f) {
```

Comment:
If we can't enforce that `style.text.lineHeight` is valid, can we add an `else` which applies a default line height?